test/src/test/java/org/corfudb/runtime/view/SequencerViewTest.java

```diff
@@ -139,4 +156,63 @@ public void checkStreamAddressSpaceAcrossEpochs() throws Exception {
                 new StreamAddressRange(streamA,  tokenCount, Address.NON_ADDRESS)).getAddressMap())
                 .isEqualTo(expectedMap);
     }
+
+    @Test
+    public void testGetStreamsId() {
+        HashSet<UUID> streamSet = new HashSet<>();
+        UUID stream1 = UUID.randomUUID();
+        UUID stream2 = UUID.randomUUID();
+        UUID stream3 = UUID.randomUUID();
+        streamSet.add(stream1);
+        streamSet.add(stream2);
+        streamSet.add(stream3);
+
+        r.getSequencerView().next(stream1).getToken();
+        r.getSequencerView().next(stream2).getToken();
+        r.getSequencerView().next(stream3).getToken();
+
+        List<UUID> streams = r.getSequencerView().getStreamsId();
+        assertThat(new HashSet<>(streams)).isEqualTo(streamSet);
+    }
+
+    @Test
+    public void testAddressSpaceTrimInBatch() {
```

Comment:
Put this test into SequencerTrimServiceTest and add more tests.

Reply:
The test is moved to SequencerTrimServiceTest.